roles/pip/defaults/main.yaml

```diff
@@ -1,5 +1,7 @@
 ---
-pip_version: "18.0"
+pip_version_default: "21.1.3"
+pip_version_centos: "20.0.1"
+pip_version: "{{ (ansible_distribution == 'CentOS') | ternary(pip_version_centos, pip_version_default) }}"
```

Comment:
Same thing, I'd recommend naming them more specifically instead of using ternary operation to control version variables.